### ISSUE TITLE
Crash with "TypeError: Cannot read property 'info' of undefined"

### ISSUE THREAD

**Issue by jonn26** [mcollina/aedes-logging#2]
Using the following minimal config with aedes-logging and a basic authentication test I am getting the below error. If I remove aedes-logging from the config it seems to work fine.

node version: 4.4.7
aedes version: 0.20.0
aedes-logging version: 1.0.0

``` javascript
'use strict'

var fs = require('fs')
var path = require('path')
var net = require('net')
var aedes = require('aedes')
var logging = require('aedes-logging')
var instance = aedes()

instance.authenticate = function (client, username, password, callback) {
    if (username == "myusername" && password.toString() == "mypassword") {
        callback(null, username === 'myusername')
    }
    else {
      var error = new Error('Auth error')
      error.returnCode = 4
      callback(error, null)
    }
}

var servers = [
  startNet()
]

logging({
  instance: instance,
  servers: servers
})


function startNet () {
  return net.createServer(instance.handle).listen(8000)
}
```

node-mini:~/aedes_script$ node aedes_auth_test_small.js
{"pid":18719,"hostname":"node-mini","level":30,"time":1470970761870,"msg":"listening","address":"::","family":"IPv6","port":8000,"protocol":"tcp","v":1}
{"pid":18719,"hostname":"node-mini","level":30,"time":1470970776333,"msg":"connected","client":{"id":"node-red"},"v":1}
{"pid":18719,"hostname":"node-mini","level":30,"time":1470970776349,"msg":"subscribed","subscriptions":[{"topic":"test","qos":0}],"client":{"id":"node-red"},"v":1}
{"pid":18719,"hostname":"node-mini","level":30,"time":1470970776354,"msg":"published","message":{"topic":"node-red/birth","qos":0,"retain":false},"client":{"id":"node-red"},"v":1}
{"pid":18719,"hostname":"node-mini","level":30,"time":1470970776354,"msg":"subscribed","subscriptions":[{"topic":"test2","qos":0}],"client":{"id":"node-red"},"v":1}
/home/jonn26/node_modules/aedes-logging/index.js:45
    client.logger.info('disconnected')
                 ^

TypeError: Cannot read property 'info' of undefined
    at Aedes.<anonymous> (/home/jonn26/node_modules/aedes-logging/index.js:45:18)
    at emitOne (events.js:77:13)
    at Aedes.emit (events.js:169:7)
    at Aedes.unregisterClient (/home/jonn26/node_modules/aedes/aedes.js:272:8)
    at finish (/home/jonn26/node_modules/aedes/lib/client.js:222:19)
    at UnsubscribeState.completeUnsubscribe (/home/jonn26/node_modules/aedes/lib/handlers/unsubscribe.js:73:5)
    at ResultsHolder.release (/home/jonn26/node_modules/aedes/node_modules/fastseries/series.js:99:22)
    at Aedes.series [as _series](/home/jonn26/node_modules/aedes/node_modules/fastseries/series.js:39:14)
    at actualUnsubscribe (/home/jonn26/node_modules/aedes/lib/handlers/unsubscribe.js:30:10)
    at handleUnsubscribe (/home/jonn26/node_modules/aedes/lib/handlers/unsubscribe.js:24:5)

**Comment by jonn26** [comment]
Hmm.. I rebooted my server and now running the same code it doesn't crash with that error anymore.

**Comment by mcollina** [comment]
Actually, this is an actual bug.

**Comment by mcollina** [comment]
Released a fix as v1.0.1.
